zmsbackend/src/Zmsbackend/ProcessSearch/Service/ProcessSearch.php

```
@@ -0,0 +1,419 @@
+        }
+    }
+
+    protected function resolveSearchReferences(
+        Entity $entity,
+        $resolveReferences
+    ): Entity {
+        if (($entity->source ?? 'active') === 'history') {
+            return $entity;
+        }
+
+        return (new \BO\Zmsbackend\Process\Service\Process())
```

PHPStan (2.2.7)

[error] Method BO\Zmsbackend\ProcessSearch\Service\ProcessSearch::resolveSearchReferences() should return BO\Zmsentities\Process but returns BO\Zmsentities\Schema\Entity.


(return.type)